### DISCUSSION TITLE
Welcome to TurboLive Discussions!

### DISCUSSION THREAD

**Discussion by AtomicInitial** [GetTurboLive/extension#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AtomicInitial** [comment]
@NotFenixio we need to get onto a codespace, or I'll just start trying to work on making the collaboration part work and commit. GIST: window.vm is how we will be able to accomplish this. eg; window.vm.loadProject(project.json), window.vm.toJSON() (which returns current JSON)

**Reply by NotFenixio** [reply]
Awesome!!

**Reply by NotFenixio** [reply]
I tried it and it works very well. But I'm not sure if scratch will catch on fire if we spam that method. We may need to buffer for some smooth, 3-second-delay realtime sliding instead of choppy, realtime "sliding".

**Reply by AtomicInitial** [reply]
i couldnt get it to work because i wasnt able to access turbowarp functions, the chrome extension blocked it and it was sandboxed from accessing the website- could you share how you did it?

**Reply by AtomicInitial** [reply]
or actually if its easier can you commit the changes?

**Reply by NotFenixio** [reply]
I tried `window.vm.toJSON()` and `window.vm.loadProject()`, but no result. by the way, mind Googling 'What's my timezone?' and sharing it? wanna make sure we catch each other online"

**Reply by AtomicInitial** [reply]
EST, also look at my previous comments and you will realize why it wont work.

**Reply by AtomicInitial** [reply]
thats the problem which is hard to get around

**Comment by NotFenixio** [comment]
ill commit my try

**Comment by AtomicInitial** [comment]
EST
